### DISCUSSION TITLE
Can publiccode.yml be used by individuals?

### DISCUSSION THREAD

**Discussion by Defkon1** [publiccodeyml/publiccode.yml#181] [Q&A]
Can publiccode.yml be used by individual developers or by public administrations only?

I work for a PA, but in my spare time I'm developing a couple of libraries that could be useful for PA's developers and projects (e.g. https://github.com/Defkon1/italian-toolkit).

Can I add the metadata file, or it is reserved to PA's projects?

Thanks in advance

**Comment by bfabio** [comment]
Hi @Defkon1, `publiccode.yml` can be used by everyone, it's [licensed under CC0-1.0](https://github.com/publiccodeyml/publiccode.yml/blob/main/LICENSE).

It also makes sense for non PAs to use it, as software not developed by PAs can still be useful to them. For example, the Italian software [catalog includes "third party software"](https://developers.italia.it/en/search?type=software_open&sort_by=release_date&page=0#136ba29d52b0295d6b844c812580640fc1e99452), each of these solutions has a `publiccode.yml` file.

You'd be very much welcome to include a `publiccode.yml` file in your repository :slightly_smiling_face:
